sandbox.config.json

```diff
@@ -1,3 +1,4 @@
 {
-    "template": "static"
+    "template": "static",
+    "hardReloadOnChange": true
```

Comment:
Cool! Wouldn't this be enough though? Or the live server?

Reply:
Eh, no, it doesn't solve the issue.

Reply:
Neither does `live-server`.

Reply:
I think we must use a different template (e.g. the one that uses `parcel` under the hood) although I still don't get why `live-server` doesn't refresh the embedded version.

Reply:
> I think we must use a different template (e.g. the one that uses `parcel` under the hood) although I still don't get why `live-server` doesn't refresh the embedded version.

Let's use `parcel` instead then: https://codesandbox.io/s/github/gomorizsolt/firebase-auth-starter/tree/fix-hot-reloading

Now this should pick up the changes right away as you edit the code (tested the embedded version as well).

Reply:
@thisismydesign https://github.com/componently-com/flying-forms-html-form-starter

This starter doesn't refresh on changes either, right?

Reply:
Please let me know if hot-reloading should also be enabled on these starters:
- https://github.com/componently-com/flying-forms-html-form-starter
- https://github.com/componently-com/algolia-starter
- https://github.com/componently-com/spotify-web-playback-starter

Reply:
Thanks! This one is enough for now.